components/Fields/Field.vue

```diff
@@ -29,7 +29,7 @@ export function isFiledEmpty(
       Object.entries(properties || {})
         .map(([key, value]) => [key.split(':'), value])
         .filter(([keys, _value]) => keys[0] === 'route' && keys.length === 3)
-        .length > 0
+        .length < 0
```

Comment:
The size of an array should never be < 0

Reply:
at least === 0 ahah sorry rookie mistake